### PR TITLE
#25322 template builder ui adjustments add padding depending on feature flag

### LINKED ISSUE #25322: Template Builder: Adjust the UI  for Perfection in Template and Layout Editor
### User Story
As a dotCMS user, I want to be able to adjust the UI of the template builder to look perfect in the Template Editor and in the Layout Editor, so that I can create a more visually appealing website.

### Acceptance Criteria
- The Template Editor and Layout Editor should look good, using the provided screenshots as reference.
- The Template Builder should be responsive down to iPad

### Proposed Objective
Core Features

### Proposed Priority
Priority 3 - Average

### External Links... Slack Conversations, Support Tickets, Figma Designs, etc.
image

### Assumptions & Initiation Needs
- The Template Builder should be compatible with all major browsers.

### Quality Assurance Notes & Workarounds
N/A

### Sub-Tasks & Estimates
- Design the UI of the Template Builder (est. 10hrs)
- Test the UI of the Template Builder (est. 8hrs)

### PR DESCRIPTION
**Pull request by danielmdob**
### <samp>🤖 Generated by Copilot at 3740ca0</samp>

Added feature flag logic and styling to the dot-template-builder component to enable or disable the new template builder UI. The feature flag is controlled by a property in the `DotPropertiesService`.

### Checklist
- [ ] Tests
- [ ] Translations
- [ ] Security Implications Contemplated (add notes if applicable)

### Additional Info
** any additional useful context or info **

### Screenshots
Original             |  Updated
:-------------------------:|:-------------------------:
** original screenshot **  |  ** updated screenshot **

### CODE REVIEW

File: core-web/apps/dotcms-ui/src/app/portlets/dot-templates/dot-template-create-edit/dot-template-builder/dot-template-builder.component.html [hunk 1]
```diff
@@ -1,4 +1,6 @@
-<p-tabView>
+<p-tabView
+    [styleClass]="(featureFlagIsOn$ | async) && 'dot-template-builder__new-template-builder'"
+>
```
Comment (zJaaal) [inline core-web/apps/dotcms-ui/src/app/portlets/dot-templates/dot-template-create-edit/dot-template-builder/dot-template-builder.component.html:3]: Can we test this?

**Review by zJaaal — CHANGES_REQUESTED** [review]

File: core-web/apps/dotcms-ui/src/app/portlets/dot-templates/dot-template-create-edit/dot-template-builder/dot-template-builder.component.html (hunk 1 above)
Comment (fmontes) [reply core-web/apps/dotcms-ui/src/app/portlets/dot-templates/dot-template-create-edit/dot-template-builder/dot-template-builder.component.html:3]: Can we?

**Review by fmontes — APPROVED** [review]

File: core-web/apps/dotcms-ui/src/app/portlets/dot-templates/dot-template-create-edit/dot-template-builder/dot-template-builder.component.spec.ts [hunk 2]
```diff
@@ -127,7 +131,7 @@ class DotTestHostComponent {
     item: DotTemplateItem;
 }
 
-describe('DotTemplateBuilderComponent', () => {
+fdescribe('DotTemplateBuilderComponent', () => {
```
Comment (rjvelazco) [inline core-web/apps/dotcms-ui/src/app/portlets/dot-templates/dot-template-create-edit/dot-template-builder/dot-template-builder.component.spec.ts:134]: Remove focused test (`fdescribe`).

File: core-web/apps/dotcms-ui/src/app/portlets/dot-templates/dot-template-create-edit/dot-template-builder/dot-template-builder.component.spec.ts [hunk 3]
```diff
@@ -166,7 +170,10 @@ describe('DotTemplateBuilderComponent', () => {
                 {
                     provide: DotPropertiesService,
                     useValue: {
-                        getKey: () => of('false')
+                        getKey: () => {
+                            console.log('running original mock');
```
Comment (rjvelazco) [inline core-web/apps/dotcms-ui/src/app/portlets/dot-templates/dot-template-create-edit/dot-template-builder/dot-template-builder.component.spec.ts:174]: Remove?

**Review by rjvelazco — CHANGES_REQUESTED** [review]

**Review by fmontes — APPROVED** [review]

**Pull request merged**